docusaurus.config.with-vnext.js

```diff
@@ -0,0 +1,17 @@
+// This file imports the config from docusaurus.config.js and adds the 'current' version to
```

Comment:
I'm thinking the best place for this file would be `./scripts`.

Reply:
Done, thanks!